scipy/datasets/_download_all.py

```diff
@@ -31,6 +31,17 @@ def download_all(path=None):
     path : str, optional
         Directory path to download all the dataset files.
         If None, default to the system cache_dir detected by pooch.
+    
+    Examples
+    --------
+    Download the datasets to the default cache location:
+
+    >>> datasets.download_all()
```

Comment:
Ah sorry, we still need one import!
```suggestion
    >>> from scipy import datasets
    >>> datasets.download_all()
```